main.py

```diff
@@ -40,10 +40,11 @@ def main():
             earned_points = executeBot(currentAccount, args)
         except Exception as e1:
             logging.error("", exc_info=True)
-            Utils.sendNotification(
-                f"⚠️ Error executing {currentAccount.username}, please check the log",
-                traceback.format_exc(),
-            )
+            if Utils.loadConfig().get("apprise", {}).get("exceptions", True):
+                Utils.sendNotification(
```

Comment:
To simplify things, we could add an optional exception parameter (defaults to None). If an exception is provided in sendNotification, we can check the config to see if sending is enabled. If it is, we send. If it isn't, we just return.